tasks/configuration/Linux.yml

```diff
@@ -90,8 +93,9 @@
     body: '{"set-user": {"solr" : "{{ solr_auth_pass }}"}}'
   become: True
   when:
+    - solr_auth_configure
     - solr_auth_type is defined
```

Comment:
Should we remove it?
`solr_auth_type is defined` in defaults so it be always True